src/pages/help-centre/components/_search-section.tsx

```diff
@@ -122,7 +122,10 @@ const SearchSection = () => {
     // putting all of the questions in a variable
     const questions = filtered_categories
         .map(({ questions }) => questions)
-        .reduce((array, questions_array) => array.concat(questions_array), filtered_categories[0].questions)
+        .reduce(
+            (array, questions_array) => array.concat(questions_array),
+            filtered_categories[0].questions,
+        )
```

Comment:
Is this change generated by eslint?

Reply:
this change is not from me. got it from merging master branch I guess.